lib/nugget/cli.rb

```diff
@@ -81,6 +81,25 @@ class CLI
       :default => false,
       :description => "Path to where results file is written/read."
 
+    option :datadog_host,
+      :long => "--datadoghost HOST",
+      :default => "localhost",
+      :description => "datadog host",
+
+    option :datadog_port,
+      :long => "--datadogport PORT",
+      :default => 8125,
```

Comment:
We run dogstatsd on port `28125` to avoid collisions/confusion with statsd (accidentally sending stuff there.)